### ISSUE TITLE
Finish the Github Wiki pages

### ISSUE THREAD

**Issue by Tresquel** [Thomasims/TeardownUMF#27]
When looking at the wiki I see this:
![image](https://user-images.githubusercontent.com/43729305/151678625-253e69c3-612b-4814-a93f-c8287449f3c7.png)

and when I try to go to one of the red tinted links it shows me this at the top:
![image](https://user-images.githubusercontent.com/43729305/151678642-6822efd5-0c1a-4158-9b96-db410efb9fda.png)

Are they still not finished and thus not public or maybe you messed up the permissions?

**Comment by Thomasims** [comment]
Yes, sorry, a red link mean the page doesn't exist. I've been procrastinating on the wiki.

**Comment by Tresquel** [comment]
Ah that's alright, should i close this issue now?

**Comment by Thomasims** [comment]
No I think it's a good reminder that I need to write those pages at some point.
